### PR TITLE
Move Window to pygame.window.Window namespace

### PR DESCRIPTION
**Pull request by Starbuck5**
Item 1 of #2603

### CODE REVIEW

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

File: buildconfig/stubs/pygame/__init__.pyi [hunk 1]
```diff
@@ -38,6 +38,7 @@ from pygame import (
     _debug as _debug,
     system as system,
     geometry as geometry,
+    window as window,
```
Comment (yunline) [inline buildconfig/stubs/pygame/__init__.pyi:41]: This file is auto generated. You need to modify the `gen_stubs.py` and rerun it.

Comment (Starbuck5) [reply buildconfig/stubs/pygame/__init__.pyi:41]: Great catch! Fixed.

**Review by yunline — APPROVED** [review]
LGTM👍

**Review by ankith26 — APPROVED** [review]
LGTM, thanks :tada:

**Pull request merged**